src/pages/quizzes.tsx

```diff
@@ -0,0 +1,281 @@
+        </QuizzesModal>
+
+        <Box px={{ base: 0, lg: 8 }} py={{ base: 0, lg: 4 }} mb={12}>
+          <Flex
+            direction={{ base: "column", lg: "row" }}
+            alignItems="start"
+            gap={{ base: 0, lg: 20 }}
+          >
+            {/* quizzes list */}
+            <Box flex={1} order={{ base: 2, lg: 1 }}>
+              <Box px={{ base: 8, lg: 0 }}>
+                <Heading
+                  fontSize={{ base: "1.75rem", lg: "2rem" }}
+                  color="body"
+                >
+                  <Translation id="basics" />
+                </Heading>
+
+                <Text mb={8} color="body">
+                  <Translation id="basics-description" />
+                </Text>
+
+                <QuizzesList
+                  content={ethereumBasicsQuizzes}
+                  quizHandler={setCurrentQuiz}
+                  nextHandler={setNextQuiz}
+                  modalHandler={setModalOpen}
+                />
+              </Box>
+
+              <Box px={{ base: 8, lg: 0 }} mb={10}>
+                <Heading
+                  fontSize={{ base: "1.75rem", lg: "2rem" }}
+                  color="body"
+                >
+                  <Translation id="using-ethereum" />
+                </Heading>
+
+                <Text mb={8} color="body">
+                  <Translation id="using-ethereum-description" />
+                </Text>
+
+                <QuizzesList
+                  content={usingEthereumQuizzes}
+                  quizHandler={setCurrentQuiz}
+                  nextHandler={setNextQuiz}
+                  modalHandler={setModalOpen}
+                />
+              </Box>
+
+              <Flex
+                direction={{ base: "column", lg: "row" }}
+                justifyContent="space-between"
+                alignItems="center"
+                bg="ednBackground"
+                borderRadius={{ base: "none", lg: "lg" }}
+                border="none"
+                p={{ base: 8, lg: 12 }}
+              >
+                <Stack mb={{ base: 4, lg: 0 }}>
+                  <Text
+                    align={{ base: "center", lg: "left" }}
+                    fontWeight="bold"
+                    color="body"
+                    mb={-2}
+                  >
+                    <Translation id="want-more-quizzes" />
+                  </Text>
+
+                  <Text align={{ base: "center", lg: "left" }} color="body">
+                    <Translation id="contribute" />
+                  </Text>
+                </Stack>
+
+                <ButtonLink
+                  to={"/contributing/learning-quizzes/"}
+                  variant="outline"
+                  hideArrow
+                  mt={0}
+                >
+                  <Flex alignItems="center">
+                    <Icon as={FaGithub} color="text" boxSize={6} me={2} />
+                    <Translation id="add-quiz" />
+                  </Flex>
+                </ButtonLink>
```

Comment:
Overflowing issues on the button

<img width="309" alt="Screenshot 2023-05-29 at 10 38 58" src="https://github.com/ethereum/ethereum-org-website/assets/62268199/dd604496-0d28-4989-abd0-22e2f7b8fde5">
<img width="422" alt="Screenshot 2023-05-29 at 10 39 07" src="https://github.com/ethereum/ethereum-org-website/assets/62268199/8cf80b90-2914-46c5-b9ab-2a27903e22c3">